### DISCUSSION TITLE
Get string representation of a tag

### DISCUSSION THREAD

**Discussion by mesvam** [cduck/drawSvg#106] [Q&A] [answered]
Is there a way to get a string representation of a single tag without adding it into a Drawing or writing out to a file?

**Comment by cduck** [comment, marked as answer]
No.  If you are willing to dig into the code you can use `element.write_svg_element(...lots of args)` like [here](https://github.com/cduck/drawsvg/blob/651c36b7a232b74a0246765589d3519780eb45c1/drawsvg/drawing.py#L279).
